src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendStep.java

```diff
@@ -176,6 +179,15 @@ public Void run() throws AbortException {
             VulnerabilityTrendHelper.logMessage(taskListener, "Checking the number of vulnerabilities for " + getBuildName());
             ContrastSDK contrastSDK = VulnerabilityTrendHelper.createSDK(teamServerProfile.getUsername(), teamServerProfile.getServiceKey(),
                     teamServerProfile.getApiKey(), teamServerProfile.getTeamServerUrl());
+
+            String applicationId = getApplicationId(contrastSDK, teamServerProfile.getOrgUuid(), getBuildName());
```

Comment:
Where is this coming from? Does it have to do with the proxy work?

Reply:
It is related to CONTRAST-21177. In the previously merged PR, I modified only the post-build step to check if the app exists. This file needs to have the same logic.